src/template/utils.js

```diff
@@ -35,8 +35,10 @@ function isBinary(type) {
  */
 function ensureUTF8Charset(resp) {
   const type = resp.headers.get('content-type');
-  if (type === 'text/html') {
-    resp.headers.set('content-type', 'text/html; charset=UTF-8');
+  if (!type) {
+    resp.headers.set('content-type', 'text/plain;charset=UTF-8');
```

Comment:
@stefan-guggisberg node-fetch used to set a content-type automatically based on the body provided in `new Response()`. see https://github.com/adobe/helix-fetch/issues

Reply:
ok, i'll have a look